libs/sdk-ui-filters/src/AttributeFilter/Components/ElementsSelect/SingleSelectionAttributeFilterElementsSelectItem.tsx

```diff
@@ -0,0 +1,49 @@
+// (C) 2023 GoodData Corporation
+import React, { useCallback } from "react";
+import cx from "classnames";
+import { IAttributeElement } from "@gooddata/sdk-model";
+import { IAttributeFilterElementsSelectItemProps } from "./types";
+import { CustomizableCheckmark, useMediaQuery } from "@gooddata/sdk-ui-kit";
+
+function getElementTitle(element: IAttributeElement) {
+    return element.title || "(empty value)";
+}
+
+/**
+ * Renders elements selection list item as a single select list item.
+ *
+ * @beta
+ */
+export const SingleSelectionAttributeFilterElementsSelectItem: React.VFC<
+    IAttributeFilterElementsSelectItemProps
+> = (props) => {
+    const { item, onSelectOnly, isSelected, fullscreenOnMobile = false } = props;
+
+    // Modify item click behavior to select only this particular item.
```

Comment:
What does mean this comment ?

Reply:
description of difference from "default" behavior